tests/smartswitch/platform_tests/test_reload_dpu.py

```diff
@@ -17,12 +17,13 @@
 from tests.platform_tests.api.conftest import *  # noqa: F401,F403
 
 pytestmark = [
-    pytest.mark.topology('t1')
+    pytest.mark.topology('smartswitch')
 ]
 
 
 def test_dpu_ping_after_reboot(duthosts, enum_rand_one_per_hwsku_hostname,
```

Comment:
enum_rand_one_per_hwsku_hostname is of scope "module" and check_smartswitch_and_dark_mode is of scope "session". You might have to modify either of it to fix scope mismatch.

Reply:
Changed check_smartswitch_and_dark_mode to be scope=module. Sorry for the confusion. I just copy pasted jinbao's suggestion. Realized now.